index.js

```diff
@@ -19,4 +21,115 @@ navLink.forEach((link) => {
     navbar.classList.toggle('hidden');
     mobileMenu.classList.add('hidden');
   });
-});
\ No newline at end of file
+});
+// #endregion
+
+// #region validate email lowercase
+const emailInput = document.querySelector('#email');
+const formtag = document.querySelector('.form2');
+
+formtag.addEventListener('submit', (event) => {
+  if (emailInput.value !== emailInput.value.toLowerCase()) {
+    event.preventDefault();
+    alert('Email must be lowercase');
+  }
+});
```

Comment:
Well done on your progress so far! 👏 It's great to see the effort you've put into your project. However, there seems to be an issue with the functionality of your form. As mentioned in the "[contact form documentation](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)," it is recommended to have a single field for the full name instead of separate fields for first name and last name. 

![image](https://github.com/Rdzaman20000/Portfolio/assets/55840999/60760f2f-6d34-4ab3-9832-36a0d47c799b)

Currently, your code includes separate fields for first name and last name in the desktop version, which is causing the form to fail when submitted. To resolve this, I recommend updating your code to remove the first name and last name fields and replace them with a single field for the full name. This will ensure a consistent user experience across both mobile and desktop versions and allow the form to function correctly.

[Document.webm](https://github.com/Rdzaman20000/Portfolio/assets/55840999/8ae76dd2-dce8-419b-bf69-d6e10c8c0195)


Take some time to review and update your code accordingly, and don't hesitate to reach out if you need any further assistance. Keep up the good work! 💪👍